dashboards/gateway-api-inference-extension/inference-extension-prometheus.json

```diff
@@ -25,61 +35,738 @@
     "columns": 48,
     "tiles": [
       {
-        "height": 4,
+        "height": 20,
+        "width": 48,
+        "widget": {
+          "title": "",
+          "id": "",
+          "text": {
+            "content": "### Charts\n#### Application Load Balancer Metrics\n\n* Tracks the request and latency of external and internal application load balancers.\n* Please refer to the [pubic page](https://cloud.google.com/load-balancing/docs/metrics) for complete list of Application Load Balancer metrics.\n\n#### Inferece Gateway Metrics\n* Tracks the performance of inference model and inference pool. \n* Please refer to the [official documentation](https://github.com/kubernetes-sigs/gateway-api-inference-extension/blob/main/site-src/guides/metrics.md) for more details of underlying metrics used in the dashboard.\n\n#### Model Server Metrics\n* Tracks the performance of model servers, including token throughput, request latency, time per output token latency and time to first token latency. \n* Charts are only populated if the model server is exporting metrics.\n* You can redirect to the detail dashboard for model servers under the integration tab (e.g. [vLLM](https://pantheon.corp.google.com/monitoring/dashboards/integration/vllm.vllm-prometheus)). \n\n### How to use:\n\n#### Inference Pool has high queue length and kv-cache utilization\n\n* **Filter**: Use the filter dropdowns above to drill down to a single cluster / location / namespace / Load Balancer.\n* **Identify issues**: Review the *Inference Pool* section for overview of high-level metrics like queue length and kv-cache utilization.\n* **Investigate**: Review *Request / s* charts.\n* **Analyze**: Examine the chart to see if the issue is caused by increasing number of requests and cause saturation of resources.\n* **Decision**: Follow the [Configure autoscaling](https://cloud.google.com/kubernetes-engine/docs/how-to/customize-gke-inference-gateway-configurations#autoscaling) to autoscales your workloads.\n\n",
+            "format": "MARKDOWN",
+            "style": {
+              "backgroundColor": "#FFFFFF",
+              "fontSize": "FS_EXTRA_LARGE",
+              "horizontalAlignment": "H_LEFT",
+              "padding": "P_EXTRA_SMALL",
+              "pointerLocation": "POINTER_LOCATION_UNSPECIFIED",
+              "textColor": "#212121",
+              "verticalAlignment": "V_TOP"
+            }
+          }
+        }
+      },
+      {
+        "height": 20,
+        "width": 48,
+        "widget": {
+          "title": "Dashboard overview",
+          "collapsibleGroup": {
+            "collapsed": false
+          },
+          "id": ""
+        }
+      },
+      {
+        "yPos": 20,
+        "height": 15,
+        "width": 16,
+        "widget": {
+          "title": "External Regional L7LB Request /s (by code)",
```

Comment:
Is it useful to have a p50 / p95 / p99 latency combined in chart (NOT broken down by code) for internal and external load balancers? 

This is something you have for Inference Model:
![Screenshot 2025-06-16 at 12 30 26 PM](https://github.com/user-attachments/assets/a6508202-541a-4a7a-872f-fb82e3bd36c2)